R/zzz.R

```diff
@@ -1,44 +1,11 @@
 # nocov start
-.onLoad <- function(...) {
-  options(warnPartialMatchDollar = TRUE)
-  Sys.setenv("_R_CHECK_LENGTH_1_CONDITION_" = "true")
+esqlabsEnv$colorPalette <- .getEsqlabsColors()
```

Comment:
@Felixmil Is it safe to do any assignments outside of `.onLoad`?

Reply:
Yes it's safe. Two reasons why it's here:
1. We need it to set this in `zzz.R` so .getEsqlabsColors() has all internal function available when this is available.
2. We cannot define it **inside** .onLoad() because that function do not have access to `esqalabsEnv`

So it is in `zzz.R` but not in `.onLoad()`